### PR TITLE
feat: adjusted sqli cloudarmor rule sensitivity level 1 to allow for cymbal bank app

### PR DESCRIPTION
**Pull request by yliaog**
…k app cookies

### CODE REVIEW

**Comment by apeabody** [comment]
flake failure, retriggering

**Review by apeabody — CHANGES_REQUESTED** [review]
Hi @yliaog - I did some testing, and I think we might be able to retain sqli-v33 if we drop the sensitivity to level 1.

**Review by apeabody — APPROVED** [review]

File: 2-multitenant/modules/env_baseline/main.tf [hunk 1]
```diff
@@ -71,9 +71,10 @@ module "cloud_armor" {
 
   pre_configured_rules = {
     "sqli_sensitivity_level_4" = {
```
Comment (apeabody) [inline 2-multitenant/modules/env_baseline/main.tf:73]: nit

```suggestion
    "sqli_sensitivity_level_1" = {
```

Comment (yliaog) [reply 2-multitenant/modules/env_baseline/main.tf]: fixed.

**Review by gtsorbo — APPROVED** [review]

**Pull request merged**